test/rebar_pkg_SUITE.erl

```diff
@@ -193,7 +193,7 @@ mock_config(Name, Config) ->
     meck:expect(rebar_dir, global_cache_dir, fun(_) -> CacheRoot end),
 
     meck:new(rebar_packages, [passthrough]),
-    meck:expect(rebar_packages, registry_dir, fun(_) -> CacheDir end),
+    meck:expect(rebar_packages, registry_dir, fun(_) -> {ok, CacheDir} end),
     meck:expect(rebar_packages, package_dir, fun(_) -> CacheDir end),
```

Comment:
It seems inconsistent `registry_dir` returns `{ok, Dir}` or `{error, Reason}` whereas `package_dir` returns `CacheDir` or crashes.